examples/generative/diffusion/training_loop.py

```diff
@@ -343,7 +351,15 @@ def training_loop(
 
         # Perform maintenance tasks once per tick.
         cur_nimg += batch_size
-        done = cur_nimg >= total_kimg * 1000
+        # done = cur_nimg >= total_kimg * 1000
+        if dataset == "dfsr":
+            done = cur_nimg >= total_kimg
+            if cur_nimg / batch_size % 500 == 0:
+                print("Progress in training iterations: loss: {}, iter: {}, cur_nimg: {}, \
```

Comment:
Use `logger0.info` instead of print statements. Example: https://github.com/NVIDIA/modulus/blob/main/examples/generative/corrdiff/train.py#L48

Reply:
The ``print`` function has been replaced with ``logger0.info``.